### DISCUSSION TITLE
Watch On: Filter Watch On services by country

### DISCUSSION THREAD

**Discussion by JoramHhub** [IgnisDa/ryot#1160] [Ideas]
Hi,

First of all, what an awesome project! Already a one to one replacement for Trakt, if not better!

I really love the Watch On feature for TV Shows and Movies, but I like to make a suggestion.
Would it be possible to create a setting to filter the services pulled from JustWatch by country or the locale set during setup?

Regards JoramH

**Comment by IgnisDa** [comment]
Just watch doesn't have an open api so this doesn't sound possible
